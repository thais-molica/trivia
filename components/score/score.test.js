import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Score from "./index";

it("Score renders correctly", () => {
  const tree = renderer.create(<Score title="easy" correct={3} error={5} />).toJSON();
  expect(tree).toMatchSnapshot();
});
