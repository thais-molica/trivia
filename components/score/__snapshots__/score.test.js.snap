// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Score renders correctly 1`] = `
Array [
  <strong
    className="jsx-318927243"
  >
    easy
  </strong>,
  <span
    className="jsx-318927243"
  >
    Correct: 
    3
  </span>,
  <span
    className="jsx-318927243"
  >
    Incorrect: 
    5
  </span>,
]
`;
